src/main/java/org/broadinstitute/consent/http/models/DuosUser.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.consent.http.models;
+
+/**
+ * This class is used to represent an authenticated user that exists as a User in the Consent
+ * system. It extends the AuthUser class which is not guaranteed to have a user associated with it.
+ */
+public class DuosUser extends AuthUser {
```

Comment:
I wonder if it's a good idea to subclass here after all. I was thinking it would help with migration, where you could pass `DuosUser` instead of `AuthUser` without changing the code. But having it as a field could work too, and then we wouldn't need to copy the data into the base class, we could embed the `AuthUser` passed to the constructor.

Reply:
My first approach was using a field. I tend to like this approach better as it opens us up for different kinds of users that might be an `AuthUser`. We don't have the use case now, but I could imagine a `ServiceAccountUser` that needs special treatment, like logging, for example.